.env.example

```diff
@@ -202,5 +215,12 @@ VITE_APP_TITLE="LibreChat"
 # and have set the GOOGLE_CLIENT_ID and GOOGLE_CLIENT_SECRET above
 VITE_SHOW_GOOGLE_LOGIN_OPTION=false
 
+# Enable OpenID Login
+# This enables/disables the Login with OpenID button on the login page.
+# Set to true if you have registered the app with an OpenID provider
+# and have set the OPENID_CLIENT_ID, OPENID_CLIENT_SECRET, OPENID_ISSUER, OPENID_SESSION_SECRET, and OPENID_SCOPE in the /api/.env file
+VITE_SHOW_OPENID_LOGIN_OPTION=false
+VITE_SHOW_OPENID_LOGIN_NAME=OpenID
```

Comment:
This one could be simplified slightly too. I think `VITE_OPENID_NAME` or `SCRIPT_OPENID_NAME` would probably work. Just slightly less characters

Reply:
Side note: do we need this at all? Is it common for people to rename OpenID to something else? I might even suggest removing this as an option and using "OpenID" instead in the code (or at the very least having it default to "OpenID" if it's left blank